react/images/DeleteIcon.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+/**
+ * Profile icon component in svg
+ */
+export default class DeleteIcon extends Component {
+  static propTypes = {
+    /* Percentage size of the icon */
+    size: PropTypes.number,
+    /* Fill color for the icon */
+    fillColor: PropTypes.string,
+  }
+
+  static defaultProps = {
+    size: 20,
+    fillColor: '#979899',
+  }
+
+  render() {
+    const { size, fillColor } = this.props
+    return (
+      <svg width={`${size}px`} height={`${size}px`} viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
+        <title>e delete</title>
+        <g fill={this.props.secondaryfill}>
```

Comment:
Put `secondaryfill` inside the destructuring statement on line 21, and also _camelCase_ it to `secondaryFill`